utils/include/sidewalk_version.h

```diff
@@ -14,7 +14,8 @@
 	LOG_PRINTK("----------------------------------------------------------------\n");          \
 	LOG_PRINTK("Sidewalk SDK        = %d.%d.%d.%d\n", SID_SDK_MAJOR_VERSION,                   \
 		   SID_SDK_MINOR_VERSION, SID_SDK_PATCH_VERSION, SID_SDK_BUILD_VERSION);           \
-	LOG_PRINTK("APP_BUILD_VERSION   = %s\n", STRINGIFY(APP_BUILD_VERSION));                    \
+	LOG_PRINTK("APP_BUILD_VERSION   = %s\n", STRINGIFY(APP_BUILD_VERSION));			   \
+	LOG_PRINTK("APP_NAME            = %s\n", CONFIG_SIDEWALK_APPLICATION_NAME);		   \
```

Comment:
Is'nt it redundant, when `CONFIG_NCS_BOOT_BANNER` is enabled? You can use `CONFIG_NCS_APPLICATION_BOOT_BANNER_STRING` (and the build version is a git commit hash)

Reply:
ok, that is a mistake, I intended to replace the APP_BUILD_VERSION with APP_NAME but I see I just added APP_NAME.

Reply:
I intentional leave APP_BUILD_VERSION as it's required for tests